patches/typedi@0.10.0.patch

```diff
@@ -1,20 +1,14 @@
 diff --git a/cjs/container-instance.class.js b/cjs/container-instance.class.js
-index e473b1e652aa0b6e7462f7ba93fcef2812483b20..1e406113d68c401ee170c997afb53e5f71edeee2 100644
+index e473b1e652aa0b6e7462f7ba93fcef2812483b20..9e57857e5584373b88a9fad3fbb37bbcc56b554a 100644
 --- a/cjs/container-instance.class.js
 +++ b/cjs/container-instance.class.js
-@@ -209,6 +209,7 @@ class ContainerInstance {
-             // this allows us to support javascript where we don't have decorators and emitted metadata about dependencies
-             // need to be injected, and user can use provided container to get instances he needs
-             params.push(this);
-+            if (process.env.NODE_ENV === 'production') Object.freeze(constructableTargetType.prototype);
```

Comment:
removed this because it wasn't being used since we can't really patch dependencies in prod using these patches.